trimmer_decorator.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'base_decorator'
+require_relative 'person'
+require_relative 'capitalize_decorator'
```

Comment:
- [ ] There is no need to import Person and CapitalizeDecorator into this file. Kindly remove these imports.